docker/run.sh

```diff
@@ -32,7 +32,9 @@ function setup() {
     fi
 
     if [ "$TEST" = "javascript" -o "$JS_SETUP" = "yes" ]; then
-        npm install npm@latest
+        curl -sL https://deb.nodesource.com/setup | sudo bash - && \
+        apt-get install -yq nodejs build-essential
+        npm install -g npm
```

Comment:
Is the next line still needed? Seems like this is doing a global install then a local one, is that accurate?

Reply:
nevermind, this was updated in a later commit